composer.json

```diff
@@ -27,16 +27,36 @@
 		}
 	},
 	"require" : {
-		"php" : ">=5.6",
-		"symfony/config" : "~2|~3|^4",
-		"symfony/yaml" : "~2|~3|^4"
+		"php" : ">=7.2",
+		"symfony/config" : "^5.0",
+		"symfony/yaml" : "^5.0"
 	},
 	"require-dev" : {
-		"phpunit/phpunit" : "^5.7",
-		"sami/sami" : "~3|~4"
+		"phpunit/phpunit": "^9.0",
```

Comment:
just `^8.0` for now.

Reply:
Sorry. I didn't see that composer installs 9.0 by default.